content/project/podcast/index.md

```diff
@@ -0,0 +1,42 @@
+---
+date: 2022-08-11
```

Comment:
Maybe this date should be set to the most recently recorded podcast to better reflect its importance in the bigger hierarchy? Otherwise it's always going to get pushed to the bottom by newer things, even when we have a new podcast out. The date is not displayed anywhere and is only used for sorting purposes.

Reply:
smart, til we sort the weighting out!